extensions/bitwarden/src/components.tsx

```diff
@@ -76,3 +77,12 @@ export function UnlockForm(props: { onUnlock: (token: string) => void; bitwarden
     </Form>
   );
 }
+
+export function CopyPasswordToClipboardAction(props: { title: string; content: string }): JSX.Element {
+  async function doCopy() {
+    const copiedSecurely = await copyPassword(props.content);
+    showHUD(copiedSecurely ? "Copied password to clipboard" : "Copied to clipboard");
```

Comment:
For better dev experience, we shouldn't rely on a boolean to check if the password was copied securely. Instead we could return a something more explicit, like `{ copiedSecurely: true }` on the `copyPassword` function.

```suggestion
    const { copiedSecurely } = await copyPassword(props.content);
    showHUD(copiedSecurely ? "Copied password to clipboard" : "Copied to clipboard");
```